datadog_checks_base/datadog_checks/utils/common.py

```diff
@@ -0,0 +1,10 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+
+from six.moves.urllib.parse import urlparse
+
+
+def get_docker_hostname():
+    return urlparse(os.getenv('DOCKER_HOST')).hostname or 'localhost'
```

Comment:
* Hmm, `None` is valid in Python 3 and not 2, good catch!
* Yes I plan to do this in a separate PR.
* I considered this but I thought it may be useful for Docker checks too. I can move it if you want.

Reply:
Yes please, let's move it